scripts/config-uart-serial.sh

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env bash
+
+# Remove any console definitions on serial0
+sed -i 's/ console=serial0,115200//' /boot/cmdline.txt
```

Comment:
What will happen if the user is using the serial with a different baudrate ? 
We are not going to remove it and add this line in the end ?
Can we use a regex to detect and remove and console=serial0* ?

Reply:
done @patrickelectric